lib/ui.js

```diff
@@ -2,18 +2,37 @@
 const path = require('path')
 const serve = require('serve-static')
 
-module.exports.serveRedoc = function serveRedoc (documentUrl) {
+module.exports.serveRedoc = function serveRedoc (documentUrl, opts) {
+  const toKebabCase = (string) =>
+    string
+      .replace(/([a-z])([A-Z])/g, '$1-$2')
+      .replace(/[\s_]+/g, '-')
+      .toLowerCase()
+  const options = JSON.parse(JSON.stringify(opts))
+  delete options.coerce
+  delete options.htmlui
+  delete options.basePath
```

Comment:
Really not a fan of this approach. We for sure should not be deep cloning with JSON, but an easier and better way to do this is to do a reduce or loop which plucks all but the fields we want to remove onto a single new object. That is not only *way* faster than JSON.parse/stringify it also means you could do the `toKebabCase` and field exclusion in one pass.

Reply:
addressed in this commit: https://github.com/wesleytodd/express-openapi/pull/47/commits/a36605409f61a1fae4e9d6507837cbd5ba9f8884